shared/src/main/java/uk/gov/di/authentication/shared/services/DynamoIdentityService.java

```diff
@@ -74,19 +78,19 @@ public void saveIdentityClaims(
             String ipvCoreIdentity) {
         var identityCredentials =
                 new IdentityCredentials()
-                        .setSubjectID(subjectID)
-                        .setAdditionalClaims(additionalClaims)
-                        .setIpvVot(ipvVot)
-                        .setIpvCoreIdentity(ipvCoreIdentity)
-                        .setTimeToExist(
+                        .withSubjectID(subjectID)
+                        .withAdditionalClaims(additionalClaims)
+                        .withIpvVot(ipvVot)
+                        .withIpvCoreIdentity(ipvCoreIdentity)
+                        .withTimeToExist(
                                 NowHelper.nowPlus(timeToExist, ChronoUnit.SECONDS)
                                         .toInstant()
                                         .getEpochSecond());
 
-        identityCredentialsMapper.save(identityCredentials);
+        dynamoIdentityCredentialsTable.putItem(identityCredentials);
```

Comment:
Now that this is doing a put rather than an update (under the hood), you can remove that pesky TODO in the `IpvCallbackHandler` 😄